string.c

```diff
@@ -7185,15 +7201,18 @@ rb_str_upcase_bang(int argc, VALUE *argv, VALUE str)
 
 /*
  *  call-seq:
- *     str.upcase              -> new_str
- *     str.upcase([options])   -> new_str
+ *    upcase(*options) -> string
+ *
+ *  Returns a string containing the upcased characters in +self+:
+ *
+ *     s = 'Hello World!' # => "Hello World!"
+ *     s.downcase         # => "hello world!"
```

Comment:
This example should use `upcase`, not `downcase`

Reply:
Done.